test/show.jl

```diff
@@ -507,10 +507,10 @@ end
 
 # PR 16221
 # Printing of upper and lower bound of a TypeVar
-@test string(TypeVar(:V, Signed, Real, false)) == "Signed<:V<:Real"
+@test string(TypeVar(:V, Signed, Real)) == "Signed<:V<:Real"
 # Printing of primary type in type parameter place should not show the type
 # parameter names.
-@test string(Array) == "Array{T,N}"
+@test string(Array) == "Array"
```

Comment:
I thought we didn't want this to change, so that you can type `Array` at the REPL and see what it is composed of.